src/components/OptionsList/BaseOptionsList.js

```diff
@@ -170,14 +170,28 @@ function BaseOptionsList({
      */
     const renderItem = ({item, index, section}) => {
         const isItemDisabled = isDisabled || section.isDisabled || !!item.isDisabled;
+        const isSelected = Boolean(
```

Comment:
You could use `_.some(...)` instead of `Boolean(_.find(...))`

Reply:
Done - 68727ae.